ArcadiaParties-SPA/src/app/user/services/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models/User';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  mockUser = {
```

Comment:
Variable should have `private` visibility modifier.

Reply:
`private` modifier added